docs/api.rst

```diff
@@ -66,3 +66,14 @@ VirtualiZarr's :py:class:`~virtualizarr.ManifestArray` objects support a limited
     stack
     expand_dims
     broadcast_to
+
+Readers
+=======
+
+.. currentmodule:: virtualizarr.readers.dmrpp
+.. autosummary::
+    :nosignatures:
+    :toctree: generated/
+
+    DMRParser
+    DMRParser.parse_dataset
```

Comment:
Do these need to be public? What's the scenario where someone actually calls these instead of just using `open_virtual_dataset(..., filetype='dmr++')`?

Reply:
Hmm yes you're right. I think I thought since it is a class on it's own it needs some documentation but I can keep all my example docs and just remove the API docs portion